concepts/classes/introduction.md

```diff
@@ -0,0 +1,94 @@
+# Introduction
+
+It is time to get to one of the core principles of C++: object-oriented programming.
+OOP is centered around classes - user-defined types of data with their own set of functions.
+We will start with the basics and will cover more advanced topics further down the syllabus tree.
+
+## Members
+
+Classes can have **member variables** and **member functions**.
+They are accessed by the **member selection** operator `.`.
+Just as variables outside of classes, it is advisable to initialize them with a value upon declaration.
+This value will then become the default for newly created objects from this class.
+
+## Encapsulation and Information Hiding
+
+Classes offer the option to restrict the usage of their members.
+The two basic cases are `private` and `public`.
```

Comment:
I was hoping the example makes it clear and I don't (yet) have to go into the nitpicking that `protected` brings, when it is introduced. You are probably right it should get a bit more explanation

Reply:
It might just be my penchant for over-explanation surfacing.  😉 

I do think saving `protected` for when you have a very clear use case would be good.  I feel like `public`/`private`/`protected` often get swirled around and confused when someone is learning them all at once.

Reply:
Better?

Reply:
Yes!  I like the change.  😄